library/accessory_workflows/build_cellranger_reference/BuildCellRangerReferenceBundle.wdl

```diff
@@ -0,0 +1,80 @@
+task BuildCellRangerReference {
+  String ref_fasta
+  String gtf_file
+  String ref_name
+
+  String docker = "quay.io/humancellatlas/secondary-analysis-cellranger"
```

Comment:
Should we add a v1.0.0 or some other tag to this repo and then reference that here (or perhaps pass in the tag from the inputs json)? Otherwise runs of this wdl may not be easily repeatable.

Reply:
That makes sense to me! Being that said, do you think I'd better also change the cellranger wdl we have for the same reason? https://github.com/HumanCellAtlas/skylab/blob/ffd17882c831da377dce3ab471c7a44683dd00f6/pipelines/cellranger/cellranger.wdl#L110

Reply:
Yes, that sounds like a good idea! Any tag is better than nothing.